forum/qa-plugin/q2a-change-username-limit/q2a-changeusernamelimit-widget.php

```diff
@@ -22,10 +22,12 @@ public function output_widget($region, $place, $themeobject, $template, $request
             $history = $this->loadHistoryFromDatabase($user);
 
             if (isset($history)) {
-                $themeobject->output('<h2>' . qa_lang('plugin_username_limit/history_title_label') . '</h2>');
+                $themeobject->output('<div class="qa-part-message-list">');
+                $themeobject->output('<h2><a name="history">' . qa_lang('plugin_username_limit/history_title_label') . '</a></h2>');
```

Comment:
Atrybut `name` na linku jest zdeprecjonowany. Jeśli już, to należy użyć `id`. Przy czym, jeśli to ma służyć do tego, aby kotwiczyć link jako fragment URL-a, to prędzej sekcja (kontener) tego ficzera powinien miec `id`.

Reply:
Wiem, zrobiłem to dokładnie tak, jak było w q2a. Gdy dodamy <a> odpowiednio się styluje.

Reply:
@ScriptyChris dodałem do a po prostu ID. ale chyba chodziło Ci o to, że nie musi być tam id tylko wyżej - w całej sekcji?